chatterbot/__init__.py

```diff
@@ -82,12 +82,12 @@ def train(self, conversation):
 
             statement = conversation[i]
 
+            database_values = self.database.find(statement)
+
             # Create an entry if the statement does not exist in the database
-            if not self.database.find(statement):
+            if not database_values:
                 self.database.insert(statement, {})
 
-            database_values = self.database.find(statement)
```

Comment:
I _think_ this was there so the search could be re-evaluated. Otherwise `database_values` is still `None` (like it was before).